aries-site/src/pages/components/toast-notifications.mdx

```diff
@@ -0,0 +1,100 @@
+</Example>
+
+## Guidance
+
+Everything that you need to know about toasts and more.
+
+### When should you use a Toast Notification?
+
+Toasts should be considered when displaying these types of information to the user:
+
+- Low attention messages that do not require user action
+- Singlular status updates
+- Confirmations
+- Information that does not need to be followed up
+
+Do not use Toasts if the information contains the following:
+
+- High attention and crtitical information
+- Time-sensitive information
+- Requires user action or input
+- Batch updates
+
+Due to their limited use cases, toasts are advised to be used sparingly or even substituted entirely for a different type of notification. In no way are toasts mandatory elements for a platform’s notification system.
+
+## Anatomy 
+
+<Box pad="small">
+  <ToastDiagram />
+</Box>
+
+Toasts are compromised of three main elements:
+
+1. Status Indicator
+2. Content (Title + Message)
+3. Close button
+
+Aside from this, all toasts have a fixed width size at Medium (384px) while the height changes based off the amount of content. 
+
+### Status Indicator
+
+Avoid using toasts for very critical information or alerts. Users may accidentally dismiss or entirely miss a toast, so it is best to avoid this case for important information. Additionally, avoid using toasts, critical or not, if it requires a user action. 
+
+The status indicators used for toasts are strictly for OK, Warning, and Unknown statuses.
+
+<Box direction="row" gap="medium" pad={{bottom: 'medium'}}>
+    <StatusGoodSmall color="status-ok" />
+    <StatusWarningSmall color="status-warning" />
+    <StatusUnknownSmall color="status-unknown" />
+</Box>
+
+You can read more guidance about status indicators [here on this page](/components/status-indicator).
+
+### Content (Title + Message)
+
+All toasts have content which is comprised of a title and a message. However, the content only requires a title, not necessarily a message. The title and message should be distinguishable from each other by the weight of the text. The status title and message should be distinguishable by the weight of the text.
+
+1. Amount of Information
+
+Toast notifications should not have too much information inside of them. Additionally, each notification should only revolve around one idea or subject. Because a toast notification is only present on the screen for a short amount of time, a user using assistive technology needs to have an understanding of what information is in the notification. Therefore, the content should be minimal and direct.
+
+For best practice, keep the amount of information to three lines or under. This three line limit either can be a combination of both the title and the message or a three-lined title. If the information is heavy and long, consider using a different type of notification other than toast. NEVER truncate text within a toast. The main point is to keep the information concise and clear.
```

Comment:
Typically, we try to avoid using all caps and just stick with sentence case.

```suggestion
For best practice, keep the amount of information to three lines or under. This three line limit either can be a combination of both the title and the message or a three-lined title. If the information is heavy and long, consider using a different type of notification other than toast. Never truncate text within a toast. The main point is to keep the information concise and clear.
```